extensions/rapids_version_templating.py

```diff
@@ -1,29 +1,89 @@
 import re
+from copy import deepcopy
+from typing import TYPE_CHECKING
 
 from docutils import nodes
 
+if TYPE_CHECKING:
+    import sphinx
+
 
 class TextNodeVisitor(nodes.SparseNodeVisitor):
```

Comment:
You mentioned renaming this, does this still need to be done?

Reply:
yes definitely, I must hav written that in the description and then got distracted before pushing it, sorry 😅 

Will do that shortly.

Reply:
updated! sorry about that